pyslackersweb/__init__.py

```diff
@@ -12,30 +10,33 @@
 from .middleware import request_context_middleware
 from . import settings, website
 
-logging.basicConfig(
-    level=logging.INFO, format="%(asctime)s - %(name)s - %(levelname)s - %(message)s"
-)
-logging.captureWarnings(True)
-logging.getLogger(__name__).setLevel(logging.DEBUG)
 
+def setup_logging() -> None:
```

Comment:
This ends up just being called anyway, why not leave it in the init to just auto-run? Same with sentry - seems useful to have it set up during migrations too